docs/cow-protocol/reference/core/tokens.md

```diff
@@ -30,3 +30,17 @@ There is an explicit bad token list, containing tokens that have been identified
 One such example is tokens that take a fee on transfer.
 The bad token list is not exhaustive and is supplemented by automated bad token detection, which may be flaky, (e.g. if a token exhibits rounding issues on transfers) depending on certain conditions.
 In any case, if a token is on the bad token list, it will result in an `UnsupportedToken` error when attempting to retrieve a quote or place an order.
+
+## New ERC-20 tokens
+
+Have you created a new `ERC-20` and want to make it seemlessly tradeable on CoW Protocol?
+
+- Bootstrap a Uni v2 (or some other well-known AMM) pool with the token (and it's minimum viable liquidity)
+- Reach out to solvers to let them know about the best liquidity source and how they can integrate it
+- Fill out the [Add Token](https://github.com/cowprotocol/token-lists/issues/new?assignees=&labels=addToken&projects=&template=addTokenForm.yml&title=%5BAddToken%5D+%60SYMBOL%60+on+%60NETWORK%60) form if you want to express your desire to have the token added to CoW Swap's token list
+
+:::caution
+
+Submitting a token to the token list does **not** guarantee that it will be added to the CoW Swap token list.
```

Comment:
I would also add:

```
Submitting a token to the token list does **not** guarantee that it will be added to the CoW Swap token list. 
Even if it's added, doesn't mean and endorsement by the team or CoW DAO.
```

Legal would be happy about that I guess.

Reply:
Tracked in #250